docs/pages/reference/mesheryctl/subcommands/mesheryctl-sys-status.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: mesheryctl system status | Meshery
+permalink: reference/mesheryctl/system/status
+type: reference
+display-title: "false"
+language: en
+lang: en
+categories: en
```

Comment:
What is “categories: en”? This this should be removed.

Reply:
This is for info IG. We can remove it though

Reply:
This categories thing exist in most of the files, we need to remove this when we overhaul the docs also 👀

Reply:
Agreed bro. We need to deal this in all files